tests/plugins/test_resolve_composes.py

```diff
@@ -289,9 +289,10 @@ def test_request_compose_for_modules(self, workflow, reactor_config_map):  # noq
 
     @pytest.mark.parametrize(('pulp_arches', 'arches', 'signing_intent', 'expected_intent'), (  # noqa:F811
         (None, None, 'unsigned', 'unsigned'),
-        (['x86_64'], None, 'release', 'beta'),
+        (['x86_64'], None, 'release', 'release'),
```

Comment:
It isn't immediately clear why this is needed.

Reply:
No arches, so no compose performed, so the expected intent won't change from the compose lookup, so it's back to what was passed in.

Reply:
Might be nice to explain that in a comment.

Reply:
Fixed

Reply:
Thanks, explanations look great.